sale_order_margin_percent/models/sale_order_margin.py

```diff
@@ -12,5 +12,8 @@ class SaleOrder(models.Model):
 
     @api.depends('margin', 'amount_untaxed')
```

Comment:
adding @api.multi would be no failure.

Reply:
On v10, @api.multi is the default

Reply:
correct - but it does not hurt - and makes it clear on a short view

Reply:
I tend to skip this as it is only needed for public methods to work with RPC in v10 and we should move the same direction as Odoo core and save lines of code for something which is default behavior. We do not add ```store=True``` to every normal field, do we?